mobile/lib/util/preferences.dart

```diff
@@ -7,6 +9,27 @@ class Preferences {
 
   static const userSeedBackupConfirmed = "userSeedBackupConfirmed";
   static const emailAddress = "emailAddress";
+  static const openPosition = "openPosition";
+
+  getOpenPosition() async {
+    SharedPreferences preferences = await SharedPreferences.getInstance();
+    return preferences.getString(openPosition);
```

Comment:
Do you need to add `?? "";` same as in `getEmailAddress`?

Reply:
Doesn't matter as it returns null in case it's not set and that would run into the default path on the loading screen.